docker-compose.yml

```diff
@@ -1,37 +1,53 @@
 version: '3.7'
+
+x-environment: &environment
+  IWFREPO: 'true'
+  IWFREPO_HTTPS: 'false'
+  IWFREPO_HOST: '192.168.18.14'
+  IWFREPO_PORT: '8087'
+  IWFREPO_INTERVAL: '300'
+  REDIS_HOST: 'redis'
+  REDIS_PORT: '6379'
+
 services:
   nginx:
-    image: nginx:latest
+    image: nginx:1.18-alpine
     container_name: nginx
     depends_on:
-      - flask
+      - flask-app
     volumes:
       - ./nginx/nginx.conf:/etc/nginx/conf.d/default.conf
     networks:
       - mso-lo-net
     ports:
       - 80:80
-  flask:
+  flask-app:
     image: 5geve/mso-lo:latest
-    container_name: mso-lo
+    container_name: flask-app
```

Comment:
You are right. thanks. I was confused by the image name assigned by Docker:
```
CONTAINER ID        IMAGE                  COMMAND                  CREATED             STATUS              PORTS                NAMES
758cd98c4cf2        mso-lo_flask-app       "uwsgi --ini app.ini"    2 hours ago         Up 5 seconds        5000/tcp             flask-app
```
As you can see, the image name gets a prefix `mso-lo` but the container name does not get it.
To avoid conflicts maybe is better to add the prefix to all `container_name` fields in the `docker-compose` files. What do you think?

Reply:
Actually, we could also remove the field since it generates errors with scaling: https://docs.docker.com/compose/compose-file/#container_name

Reply:
By removing the `container_name` fields I obtain this:
```
CONTAINER ID        IMAGE                  COMMAND                  CREATED             STATUS              PORTS                NAMES
d9a17271571f        nginx:1.18-alpine      "/docker-entrypoint.…"   10 seconds ago      Up 9 seconds        0.0.0.0:80->80/tcp   mso-lo_nginx_1
5b93572e0c37        mso-lo_flask-app       "uwsgi --ini app.ini"    10 seconds ago      Up 9 seconds        5000/tcp             mso-lo_flask-app_1
```
That is actually what we wanted I think.